src/starknet-wrappers.ts

```diff
@@ -127,6 +140,15 @@ export abstract class StarknetWrapper {
             prepared.push("--token", options.token);
         }
 
+        if (options.sender) {
+            prepared.push("--sender", options.sender);
+        }
+
+        if (options.maxFee == null) {
+            throw new StarknetPluginError("No maxFee provided for declare tx");
+        }
+        prepared.push("--max_fee", options.maxFee);
```

Comment:
Thank you for adding this!

Reply:
A new version should be released today. If we manage to get all the tests passing, it will be 0.6.6, otherwise it will be 0.6.6-alpha.4

Reply:
Cool, looking forward to it!